mesheryctl/internal/cli/root/system/start.go

```diff
@@ -118,7 +118,7 @@ func ApplyManifestFiles(manifestArr []utils.Manifest, requestedAdapters []string
 		adapterDeployment := adapterFile + "-deployment.yaml"
 		adapterService := adapterFile + "-service.yaml"
 
-		if utils.IsAdapterValid(manifestArr, adapter+"-deployment.yaml") == false {
+		if !utils.IsAdapterValid(manifestArr, adapter+"-deployment.yaml") {
```

Comment:
Neat 👍🏼 

IDK why I felt `== false` would be more readable. Maybe because I always overlook the `!` and I got into a habit of making it obvious by adding the `==`